secure/key-manager/src/lib.rs

```diff
@@ -19,6 +19,7 @@
 //! KeyManager talks to its own storage through the `LibraSecureStorage::Storage trait.
 #![forbid(unsafe_code)]
 
+use crate::libra_interface::LibraInterface;
```

Comment:
maybe re-export?

Reply:
I was thinking I could just leave that until someone actually needs it?

Reply:
Well I guess in practice only the binary will need any of this crate, no?